pid_gains: { kp: 0.45, ki: 0.0, kd: 0.3 }
wheel_base: 0.64
max_steer_ang: 1.309
ros_control: True

